src/webui/www/private/scripts/client.js

```diff
@@ -928,6 +928,21 @@ window.addEventListener("DOMContentLoaded", () => {
 
         $("freeSpaceOnDisk").textContent = "QBT_TR(Free space: %1)QBT_TR[CONTEXT=HttpServer]".replace("%1", window.qBittorrent.Misc.friendlyUnit(serverState.free_space_on_disk));
 
+        const lastExternalAddressV4 = serverState.last_external_address_v4;
+        const lastExternalAddressV6 = serverState.last_external_address_v6;
+        const hasIPv4Address = lastExternalAddressV4 !== "";
+        const hasIPv6Address = lastExternalAddressV6 !== "";
+        let lastExternalAddressLabel = "QBT_TR(External IP: N/A)QBT_TR[CONTEXT=HttpServer]";
+        if (hasIPv4Address || hasIPv6Address) {
+            if (hasIPv4Address && hasIPv6Address)
+                lastExternalAddressLabel = "QBT_TR(External IPs: %1, %2)QBT_TR[CONTEXT=HttpServer]";
+            else
+                lastExternalAddressLabel = "QBT_TR(External IP: %1%2)QBT_TR[CONTEXT=HttpServer]";
+        }
+        // replace in reverse order ('%2' before '%1') in case address contains a % character.
```

Comment:
I don't think it would help. In fact you should do the opposite since only ipv6 could contain `%` char. Therefore, put in ipv4 first and ipv6 second.
```js
$("externalIPs").textContent = lastExternalAddressLabel.replace("%1", lastExternalAddressV4).replace("%2", lastExternalAddressV6);
```

Reply:
Since we're replacing `%2` with the IPv6 address first, the intended `%1` will always be the first `%` sequence in the string. For example, the string will be `External IPs: %1{some IPv6 address}`